docs/content/troubleshooting.md

```diff
@@ -5,47 +5,90 @@
 
 We also maintain a [known issues page]({{urlRoot}}/known-issues) for any open bugs in the SpatialOS GDK for Unity.
 
-**The following exception is thrown: `<name of a type>` is an IComponentData, and thus must be blittable
-(No managed object is allowed on the struct).**
+
+#### Error reported
+When building workers (Unity Editor menu: **Build For Cloud** > **my_worker_name**):<br/>
+ _Error building player because build target was unsupported_
+
+**Issue and workarounds**<br/>
+ You don't have linux build support enabled. You need this because all server-workers in a cloud deployment run in a Linux environment. 
```

Comment:
depending on the build target it could also be another build support (e.g. mac). Maybe need to rephrase it a bit that they should check their build configuration and that cloud builds need to be linux

Reply:
OK